pkg/apis/aws/v1alpha1/accountpool_types.go

```diff
@@ -16,9 +16,22 @@ type AccountPoolSpec struct {
 // AccountPoolStatus defines the observed state of AccountPool
 // +k8s:openapi-gen=true
 type AccountPoolStatus struct {
-	PoolSize          int `json:"poolSize"`
+	PoolSize int `json:"poolSize"`
+
+	// UnclaimedAccounts is an approximate value representing the amount of non-failed accounts
 	UnclaimedAccounts int `json:"unclaimedAccounts"`
-	ClaimedAccounts   int `json:"claimedAccounts"`
+
+	// ClaimedAccounts is an approximate value representing the amount of accounts that are currently claimed
+	ClaimedAccounts int `json:"claimedAccounts"`
+
+	// AvailableAccounts denotes accounts that HAVE NOT BEEN CLAIMED and are NOT reused and are READY to be claimed.  (This differs from the UnclaimedAccounts above which is all non-failed accounts, this number is only READY accounts)
```

Comment:
I found the comparison to UnclaimedAccounts accounts was missing the part where they too have never been claimed. Here's an attempt at improving it, but might need more cleanup - wording is hard.
```suggestion
	// AvailableAccounts denotes accounts that HAVE NEVER BEEN CLAIMED, so NOT reused, and are READY to be claimed.  This differs from the UnclaimedAccounts, who similarly HAVE NEVER BEEN CLAIMED, but include ALL non-FAILED states
```